whisper/stt/processing/utils.py

```diff
@@ -65,11 +68,8 @@ def get_language(language = None):
         language = {v: k for k, v in LANGUAGES.items()}.get(language.lower(), language)
         # Raise an exception for unknown languages
         if language not in LANGUAGES:
-            available_languages = (
-                list(LANGUAGES.keys())
-                + [k[0].upper() + k[1:] for k in LANGUAGES.values()]
```

Comment:
Why did you remove capitalization ?
I think I did that for some reason.
Did you tested complete language names : how it is sensitive to the case ("French", "french", "FRENCH", ...) ?

Reply:
This `available_languages` only purpose is for the error message so it doesn't break anything to remove the capitalization.

Reply:
OK, it will just print "french" instead of "French".
I think I wanted to differenciate the complete language names from codes ("fr") for more readability.
But I don't really care more than that.